tasks/publish.yaml

```diff
@@ -14,50 +17,62 @@ tasks:
+
+          # dont publish arm64 for registry1 since IB images are only amd64
+          if [ "${FLAVOR}" != "registry1" ]; then            
+            zarf package publish build/zarf-package-core-istio-arm64-${VERSION}.tar.zst ${TARGET_REPO}
+            zarf package publish build/zarf-package-core-arm64-${VERSION}.tar.zst ${TARGET_REPO}
+          fi
 
   - name: bundles
     description: "Build and publish UDS bundles"
     actions:
       - description: "Create the UDS Core Standard Bundle"
         cmd: |
-          set -e        
+          set -e
           ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
           ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
 
       - description: Create the UDS Core Istio Only Bundle
         cmd: |
-          set -e        
+          set -e
           ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml k3d-istio-bundle --no-progress
           ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml k3d-istio-bundle --no-progress
 
       - description: "Publish the bundles"
         cmd: |
-          set -e        
-          uds publish bundles/k3d-standard/uds-bundle-k3d-core-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
-          uds publish bundles/k3d-standard/uds-bundle-k3d-core-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
-          uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
-          uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles --no-progress
+          set -e
+          uds publish bundles/k3d-standard/uds-bundle-k3d-core-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
+          uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
+          
+          # dont publish arm64 for registry1 since IB images are only amd64
+          if [ "${FLAVOR}" != "registry1" ]; then            
+            uds publish bundles/k3d-standard/uds-bundle-k3d-core-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
+            uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-arm64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
+          fi
 
       - description: "Tag the latest bundles"
         cmd: |
           set -e
-          pkgPath="ghcr.io/defenseunicorns/packages/uds/bundles/k3d-core"
-          zarf tools registry copy ${pkgPath}:${VERSION}-arm64 ${pkgPath}:arm64
+          pkgPath="ghcr.io/defenseunicorns/packages/uds/bundles/${FLAVOR}/k3d-core"
+          # dont tag arm64 for registry1 since IB images are only amd64
+          if [ "${FLAVOR}" != "registry1" ]; then 
+            zarf tools registry copy ${pkgPath}:${VERSION}-arm64 ${pkgPath}:arm64
```

Comment:
I'm not totally sure we really need "latest" shortcuts for the other flavors besides upstream since these are really just meant for quick testing/demo

Reply:
I dont really think we need them either way. I was just updating the publish step to have parity with the what was already happening.

IMO doing upstream one way and the other a different way is going to confuse someone trying to consume.

Reply:
k we can save for another time to think about that